presidio-cli/Pipfile

```diff
@@ -0,0 +1,15 @@
+[packages]
+presidio-analyzer = ">=2.2"
```

Comment:
CI currently fails on the allow list functionality, and the locked presidio-analyzer version is 2.2.24 (allow-list was introduced in 2.2.29). If this is causing issues we can start without the allow-list, or we can mare the version to be >=2.2.29.

Reply:
Updated to `>= 2.2.29`.